src/Defra.PTS.Web.UI/Views/TravelDocument/Partials/_ApplicationsList.cshtml

```diff
@@ -41,9 +41,21 @@
                         </td>
                         <td class="govuk-table__cell">
                             <strong class="govuk-tag @statusCssClass">
-                                @(application.Status == @AppConstants.ApplicationStatus.APPROVED 
-                                ? @Localizer["Approved"] 
-                                : @Localizer["Pending"])
+                                @if (application.Status == @AppConstants.ApplicationStatus.APPROVED){
+                                    @Localizer["Approved"]
+                                }
```

Comment:
Revert this file please, sorry forgot to remove this before giving to you.

Reply:
Reverted